app/views/tickets/show.html.haml

```diff
@@ -0,0 +1,57 @@
+%script{:src => "//maps.google.com/maps/api/js?key=#{ENV['MAPS']}"}
+%script{:src => "//cdn.rawgit.com/mahnunchik/markerclustererplus/master/dist/markerclusterer.min.js"}
+%script{:src => "//cdn.rawgit.com/printercu/google-maps-utility-library-v3-read-only/master/infobox/src/infobox_packed.js", :type => "text/javascript"}
+
+%table
+  %thead
+    %tr.table_head_color
+      %th RequestNumber
+      %th SequenceNumber
+      %th RequestType
+      %th PrimaryServiceAreaCode
+      %th AdditionalServiceAreaCodes
+      %th DigsiteInfo
+      %th ResponseDueDateTime
+  %tbody
+    %tr
+      %td= @ticket.RequestNumber
+      %td= @ticket.SequenceNumber
+      %td= @ticket.RequestType
+      %td= @ticket.PrimaryServiceAreaCode
+      %td= @ticket.AdditionalServiceAreaCodes
+      %td= @ticket.DigsiteInfo
+      %td= @ticket.ResponseDueDateTime
+%br
+#map{:style => "width: 800px; height: 400px;"}
+:javascript
+  handler = Gmaps.build('Google');
+  handler.buildMap({ provider: {}, internal: {id: 'map'}}, function(){
+      markers = handler.addMarkers(#{raw @hash.to_json});
+      handler.bounds.extendWith(markers);
+      handler.fitMapToBounds();
+  });
+
+%h1
+  Excavator data:
+%br
+%table
+%thead
+  %tr.table_head_color
+    %th CompanyName
+    %th Street
+    %th City
+    %th State
+    %th Zip
+    %th Type
+    %th CrewOnSite
+%tbody
+  %tr
+    - unless @ticket.excavator.nil?
```

Comment:
How many times i should repeat this
```ruby
- if @ticket.excavator
```
in this case if `@ticket.excavator` nil code will go to `else` statement